internal/install/static_snapshot_yml.go

```diff
@@ -89,10 +89,12 @@ services:
     environment:
     - "FLEET_ENROLL=1"
     - "FLEET_ENROLL_INSECURE=1"
+    - "FLEET_SERVER_INSECURE_HTTP=1"
     - "FLEET_INSECURE=1"
+    - "FLEET_SERVER_ENABLE=1"
```

Comment:
@blakerouse @ruflin @nchaulet I watched the observability demo session and @blakerouse's presentation (good job!) about running the agent with fleet server in a container and I'm confused about available configuration options. Recently I've removed FLEET_URL and KIBANA_HOST from global envs, but I saw @blakerouse used them. Unfortunately with current blockers it's hard to determine the correct set.

Shall I ask you to review both `kibana.config.yml` and `snapshot.yml` and recommend the best configuration for 7.13?

Reply:
@mtojek `FLEET_URL` should not be required, but at the moment there is an issue that requires it. I am working to solve this issue. As for the `KIBANA_HOST` that is required if you want the container to perform the setup of Kibana working with Fleet, which when running with `FLEET_SERVER_ENABLED` is required, unless you are running at some hook on the Kibana container instead.

Reply:
> FLEET_URL should not be required, but at the moment there is an issue that requires it. I am working to solve this issue.

Would you mind linking this issue for tracking purposes?

> As for the KIBANA_HOST that is required if you want the container to perform the setup of Kibana working with Fleet, which when running with FLEET_SERVER_ENABLED is required, unless you are running at some hook on the Kibana container instead.

In this setup we've a long running Elastic cluster (Elasticsearch, Kibana, Package Registry) and two agents (so far):
1. Separate agent's container which is used for system tests (reassigning policies).
2. Agent's container in the Kubernetes cluster.

which gives us two agent containers, each one with own FleetServer inside. Can Kibana handle it without any problems?

Reply:
Two fleet-server should just work. If not, please let us know.